core/templates/domain/skill/WorkedExampleObjectFactorySpec.ts

```diff
@@ -44,7 +44,7 @@ describe('Worked example object factory', () => {
 
   it('should create a new worked example from a backend dictionary', () => {
     let workedExample =
-          workedExampleObjectFactory.createFromBackendDict(workedExampleDict);
+           workedExampleObjectFactory.createFromBackendDict(workedExampleDict);
```

Comment:
Why this chnage?

Reply:
@DubeySandeep  actually i used eslint --fix directory path to fix the linting issues and this did all the changes like this. I think indentation was not proper and thats why it fixed it. If all these changes will lead to conflicts, I will remove all the unexpected changes.  Thanks!